src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'app';
+  title:string = "SwiftFest Slack Channel";
```

Comment:
Why it's not in a bundle?

Reply:
This is not even used, but I have placed it in the i18n/en.json file